src/backend/__mocks__/electron.ts

```diff
@@ -28,8 +32,23 @@ class Notification {
 }
 
 class BrowserWindow {
-  public constructor() {
-    return {}
+  static windows: BrowserWindow[] = []
+  options: BrowserWindowConstructorOptions = {}
+
+  constructor(options: BrowserWindowConstructorOptions) {
+    this.options = options
+  }
+
+  static getAllWindows() {
+    return this.windows
+  }
+
+  static setAllWindows(windows: BrowserWindow[]) {
+    this.windows = windows
+  }
+
+  public getOptions() {
+    return this.options
```

Comment:
with this we can make assertions about the options passed to the `new BrowserWindow` constructor, even though it's not a real method of a BrowserWindow